twilight-http/src/request/channel/thread/create_forum_thread/mod.rs

```diff
@@ -116,10 +116,8 @@ impl<'a> CreateForumThread<'a> {
             Err(source) => ResponseFuture::error(source),
         }
     }
-}
 
-impl TryIntoRequest for CreateForumThread<'_> {
-    fn try_into_request(mut self) -> Result<Request, HttpError> {
+    fn try_into_request(mut self) -> Result<Request, Error> {
```

Comment:
What is the reason for this change

Reply:
Request buildrs without a public `exec()` method or a `IntoFuture` implementation should also not implement `TryIntoRequest`. In this case calling `TryIntoRequest` would always produce an *invalid* request since the REST call requires a message (which one sets by calling the `message` method). Note that `CreateForumThreadMessage` implements both `IntoFuture` and `TryIntoRequest`.